test/parallel/test-repl.js

```diff
@@ -71,7 +71,7 @@ function clean_up() {
 function strict_mode_error_test() {
   send_expect([
     { client: client_unix, send: 'ref = 1',
-      expect: /^ReferenceError:\sref\sis\snot\sdefined\n\s+at\srepl:1:5/ },
+      expect: /^ReferenceError:\sref\sis\snot\sdefined\n/ },
```

Comment:
Is this the new end of the error? If yes, we might mark that in the RegExp and if not, I would just replace it with the new `at`.